sources/settings-extensions/oci-hooks/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+/// Settings related to host-provided OCI Hooks
+use bottlerocket_settings_sdk::{GenerateResult, SettingsModel};
+use model_derive::model;
+use std::convert::Infallible;
+
+#[model(impl_default = true)]
+pub struct OciHooksSettingsV1 {
```

Comment:
Should we add documentation that this doesn't do anything? I guess that is documented elsewhere?

Reply:
It's documented here https://bottlerocket.dev/en/os/1.19.x/api/settings/oci-hooks/. Not sure if it's documented elsewhere. I can add following for clarity: `The log4j hotpatch functionality is no longer included in Bottlerocket as of v1.15.0. The setting still exists for backwards compatibility.`